lib/FrontendTool/FrontendTool.cpp

```diff
@@ -1037,9 +1052,10 @@ static void setPrivateDiscriminatorIfNeeded(IRGenOptions &IRGenOpts,
     IRGenOpts.DWARFDebugFlags += (" -private-discriminator " + PD.str()).str();
 }
 
-static bool serializeSIB(FrontendOptions &opts, SILModule *SM,
-                         ASTContext &Context, ModuleOrSourceFile MSF) {
-  const std::string &moduleOutputPath = opts.ModuleOutputPath;
+static bool serializeSIBIfNeeded(FrontendOptions &opts, SILModule *SM,
```

Comment:
Looks like the wrong name snuck back into this version.

Reply:
Thanks. Fixed.